pipeline/src/types/prismic/exhibitionHighlightTours.ts

```diff
@@ -23,7 +21,7 @@ type WithExhibitionHighlightTourSlices = {
 export type ExhibitionHighlightTourPrismicDocument = prismic.PrismicDocument<
   {
     related_exhibition: prismic.ContentRelationshipField<
-      ExhibitionPrismicDocument,
+      'exhibitions',
```

Comment:
`ContentRelationshipField` expects a `string` (`TypeEnum - Type API ID of the document.`) and TS complained, so amended it in both places.